docs/content/guides/dagster/recommended-project-structure.mdx

```diff
@@ -13,15 +17,13 @@ At a high level, here are the aspects we'd like to optimize when structuring a c
 - You can quickly find the relevant code regardless of your familiarity with the related business logic.
 - You can organize at your own pace when you feel things have grown too big, but not over-optimize too early.
 
-As your experience with Dagster grows, you may find certain aspects of this guide no longer apply to your use cases and you may want to change the structure to adapt to your business needs. We believe every team is unique and every team finds its own way to collaborate and be productive in the end.
-
-We're constantly learning -- If you have anything you'd like to add about project layouts, we invite you to join this [GitHub Discussion](https://github.com/dagster-io/dagster/discussions/8972) to share how you organize your Dagster code.
+As your experience with Dagster grows, certain aspects of this guide no longer apply to your use cases, and you may want to change the structure to adapt to your business needs.
```

Comment:
```suggestion
As your experience with Dagster grows, certain aspects of this guide might no longer apply to your use cases, and you may want to change the structure to adapt to your business needs.
```

🔧 Adding `might` in here - we don't know if it'll apply or not. (Apologies if this was from a suggestion of mine and I missed adding this word in)